src/components/WorkspaceDashboardTaskRunsCard.vue

```diff
@@ -237,17 +240,20 @@
 }
 
 .workspace-dashboard-task-runs-card__chart-container { @apply
-  relative
+  absolute
   min-h-0
   h-16
+  left-0
+  right-0
+  bottom-4
 }
 
 .workspace-dashboard-task-runs-card__chart { @apply
   absolute
   min-h-0
-  -left-8
+  left-0
   top-0
-  -right-8
+  right-0
```

Comment:
good catch. It was always intended to span the width of the card and not to be wider than the card itself.